### ISSUE TITLE
Text spacing not !important does not account for actual font-size [78fd32, 24afc2, 9e45ec]

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-rules.github.io#1688]
This issue applies to the following three rules:
- [Letter spacing in style attributes is not !important](https://act-rules.github.io/rules/24afc2)
- [Line height in style attributes is not !important](https://act-rules.github.io/rules/78fd32)
- [Word spacing in style attributes is not !important](https://act-rules.github.io/rules/9e45ec)

This is a false positive. The rule fails the `div` for not having a letter-spacing that's less than 0.15 times the font-size, but then the `p` element reduces the font-size, and so in actuality the text ends up with letter spacing that is sufficient.

```html
<div style="font-size: 16px; letter-spacing: 2px !important">
  <p style="font-size: 10px;">
    The toy brought back fond memories of being lost in the rain forest.
  </p>
</div>
```

**Comment by Jym77** [comment]
Great catch!

It's going to be annoying to correctly express…
🤔 
```
**wide enough**: for each text node descendant whose parent's cascaded `letter-spacing` is the one specified in the `style` attribute of the target, the computed value of its [=the parent of text node] `letter-spacing` property is at least 0.12 times the computed value of its [=the parent of text node] `font-size` property; or
```
In your example, the text node's parent's cascaded `letter-spacing` is the one defined in the `div`'s `style`, but it is 0.12 times the text node's parent's computed `font-size` (`10px`), so it's OK…

This formulation should also catch cases where the `p` inside the `div` specifies another `letter-spacing`.

**Comment by Jym77** [comment]
Working on this (and #1687) on Alfa's doc, this is what I ended up with:
Replace the current
```markdown
For each test target, at least one of the following is true:

- **not important**: the [computed][] value of its [letter-spacing][] property is not [important][]; or
- **wide enough**: the [computed][] value of its [letter-spacing][] property is at least 0.12 times the [computed][] value of its [font-size][] property; or
- **cascade**: the [cascaded][] value of its [letter-spacing][] property is not a value [declared][] in its `style` attribute.
```
by
```markdown
For each test target, at least one of the following is true:

- **not important**: the [computed][] value of its [letter-spacing][] property is not [important][]; or
- **wide enough**: for each non-[whitespace][] only text node descendant of this target, either its parent is [wide enough][#wide-enough], or its parent's [cascaded][] [letter-spacing][] property is not the one [declared][] in the `style` attribute of the target.

An element has a <dfn id="wide-enough">wide enough letter spacing</dfn> if the computed value of its [letter-spacing][] property is at least 0.12 times the computed value of its [font-size][] property.
```

There is an annoying part with defining "wide enough" just after the expectation. I think this is better than trying to inline that in the bullet list. It's not really possible to have it as separate definition (or rather, we'd need one for each of the three rules since they are about different properties, and that would be defs used in one rule each, which is a bit annoying).
